src/pipecat/processors/frame_processor.py

```diff
@@ -124,13 +134,16 @@ def report_only_initial_ttfb(self):
     def can_generate_metrics(self) -> bool:
         return False
 
+    def _model_name(self) -> str | None:
+        return self._model if hasattr(self, "_model") else None
```

Comment:
I'm not sure this is a good idea. We don't want to force people to remember they have to use `_model` as a variable name. I believe it might be better to just optionally pass the model to the `FrameProcessor` functions. Just being more explicit about it.